src/style/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star--active {
```

Comment:
check this point
there is should be modifiers to change number of active stars
<img width="740" alt="image" src="https://user-images.githubusercontent.com/102411130/221791401-08039724-82d8-4203-b229-d82a763680de.png">